composer.json

```diff
@@ -1,28 +1,31 @@
 {
-    "name": "dasprid/container-interop-doctrine",
```

Comment:
Why was the formatting of this file changes? This is auto-generated by composer and thus should not have that many changes.

Reply:
I added manually the suggest section in eclipse. After I have pushed it to git, the formatting was broken, so I formatted the composer.json correct.

Reply:
Apparently your IDE added a lot of tabs in here. Please revert the formatting changes.